README.md

```diff
@@ -27,42 +34,64 @@ Or clone the latest version from Github and install it via Python setuptools:
 
 # Supported Modules
 
-The following modules are supported by PyQt5-stubs:
-
-* QtCore
-* QtWidgets
-* QtGui
-* QtDBus
-* QtNetwork
-* QtOpenGL
-* QtPrintSupport
-* QtSql
-* QtTest
-* QtXml
-* sip
-
-# Building upstream stubs
-The Dockerfile is used to build all of the stubs for the upstream PyQt5 modules.
-The Dockerfile consists of multiple build layers:
-* core: `PyQt5`
-* `PyQt3D`
-* `PyQtChart`
-* `PyQtDataVisualization`
-* `PyQtPurchasing`
-* `PyQtWebEngine`
-* an output layer
-
-Each module build layer deposits its stub files within `/output/` in its
-filesystem. The output layer then collects the contents of each into its own 
-`/output/` dir for export to the host computer. Build args are provided to 
-change the version of each module.
-
-A convenience script, `build_upstream.py`, is provided. It builds the stubs and 
-copies them to the host computer. Make sure you install `docker-py` to use it.
-It builds `$PWD/Dockerfile` (overridden with `--dockerfile`) and outputs the
-stubs to `$PWD/PyQt5-stubs` (overridden with `--output-dir`). 
-
-\* There are a few missing modules: `QtAxContainer`, `QtAndroidExtras`, 
-`QtMacExtras`, and `QtWindowsExtras`. The current project understanding is that
-they need to be built on the target platform, something a Linux-based docker
-image cannot do. The deprecated `Enginio` module is also missing.  
+The modules supported by PyQt5-stubs include modules from the PyQt5 package as well as modules from the other
+packages released by Riverbank Computing (PyQt3D, PyQtCharts, ...). Here is the full list
+of packages and modules:
```

Comment:
It doesn't strike me as worth maintaining this big list.  It seems more useful to remove the list entirely and the user can look at the source to see the present state of provided modules.